blockstack/explorer
# fix: add network modal

## Description

This PR fixes the `Add a network` modal that seemed to break with our switch to jotai (not totally sure why).

For details refer to issue #532 

Working now...
![Screen Shot 2021-09-15 at 4 10 25 PM](https://user-images.githubusercontent.com/6493321/133510301-89bcc410-ae58-4b75-ad70-0abfb05a7d5e.png)

## Type of Change

## Testing

- Go to network menu and `Add a network` - confirm modal launches successfully
- Proceed to add a new network - confirm the network is added successfully and network switches automatically
- Change back to default network and confirm this works correctly
- Try changing the `chain=` in the url to the opposite network - confirm switches network correctly (and in menu)
- Use trash icon in network settings menu to remove the custom network - confirm works correctly
